server/src/config.ts

```diff
@@ -80,6 +80,10 @@ const privateKey = readFileSync(sshKeyPath, {
 
 // helper function to parse github PEM
 const formatGithubPem = (pem: string) => {
+  if (!pem) {
```

Comment:
@pradel added this in the end as using `undefined` in config below was causing issues